heliopy/data/custom/units_dict.py

```diff
@@ -0,0 +1,33 @@
+"""
+CDF Unit Dictionary
+-------------------
+Method to obtain the unit denoted by the strings
+inside the CDF files in the UNIT attribute.
+"""
+
+import astropy.units as u
+from collections import OrderedDict
+
+
+def get(unit_string):
+    units = OrderedDict([('ratio', u.dimensionless_unscaled),
+                        ('#/cc', u.cm**-3),
+                        ('#/cm^3', u.cm**-3),
+                        ('cm^{-3}', u.cm**-3),
+                        ('ionic charge', u.electron),
```

Comment:
Does `u.electron` give a negative number? If so this should be `u.electron`.

Reply:
It doesn't give a negative number, I think we should define something else for this.

Reply:
Is the ionic charge here referring to the charged state? Like O^- or H3O+?